packages/hs-react-ui/src/components/Text/Text.stories.tsx

```diff
@@ -81,8 +84,8 @@ export const ThemedText = () => (
     StyledContainer={ThemedContainer}
     size={text('size', '1rem')}
     color={color('color', colors.grayDark)}
-    iconPrefix={select('iconPrefix', options, options.none)}
-    iconSuffix={select('iconSuffix', options, options.none)}
+    iconPrefix={getIconPrefixPath(select('iconPrefix', options, options.mdiComment))}
+    iconSuffix={getIconSuffixPath(select('iconPrefix', options, options.mdiComment))}
```

Comment:
again this knob should be iconSuffix
```suggestion
    iconSuffix={getIconSuffixPath(select('iconSuffix', options, options.mdiComment))}
```